[2.8.0 deployment] Directory handling in deployment workflow does not normalize names

From the deploy action:

  mkdir -p testsuite/dist
  mv dist/MDAnalysisTests-* testsuite/dist
  shell: /usr/bin/bash -l {0}
mv: cannot stat 'dist/MDAnalysisTests-*': No such file or directory
Error: Process completed with exit code 1.

The action needs to try to move dist/mdanalysistests-* instead (setuptools now normalizes package names to not have capital letters).

--- a/.github/workflows/deploy.yaml
+++ b/.github/workflows/deploy.yaml
@@ -139,7 +139,7 @@ jobs:
       - name: move_test_dist
         run: |
           mkdir -p testsuite/dist
-          mv dist/MDAnalysisTests-* testsuite/dist
+          mv dist/mdanalysistests-* testsuite/dist
 
       - name: upload_source_and_wheels
         uses: pypa/gh-action-pypi-publish@v1.11.0
@@ -168,7 +168,7 @@ jobs:
       - name: move_test_dist
         run: |
           mkdir -p testsuite/dist
-          mv dist/MDAnalysisTests-* testsuite/dist
+          mv dist/mdanalysistests-* testsuite/dist
 
       - name: upload_tests
         uses: pypa/gh-action-pypi-publish@v1.11.0
@@ -198,7 +198,7 @@ jobs:
       - name: move_test_dist
         run: |
           mkdir -p testsuite/dist
-          mv dist/MDAnalysisTests-* testsuite/dist
+          mv dist/mdanalysistests-* testsuite/dist
 
       - name: upload_source_and_wheels
         uses: pypa/gh-action-pypi-publish@v1.11.0
@@ -224,7 +224,7 @@ jobs:
       - name: move_test_dist
         run: |
           mkdir -p testsuite/dist
-          mv dist/MDAnalysisTests-* testsuite/dist
+          mv dist/mdanalysistests-* testsuite/dist
 
       - name: upload_tests
         uses: pypa/gh-action-pypi-publish@v1.11.0

